### ISSUE TITLE
CTRL-; does the same as CTRL-G in Windows Terminal

### ISSUE THREAD

**Issue by Drllap** [neovim/neovim#21184]
### Neovim version (nvim -v)

NVIM v0.8.1
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compiled by runneradmin@fv-az30-11

Features: -acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM\sysinit.vim"
  fall-back for $VIM: "C:/Program Files (x86)/nvim/share/nvim"

Run :checkhealth for more info

### Vim (not Nvim) behaves the same?

no, vim 9.0.0916

### Operating system/version

Windows 10

### Terminal name/version

Windows Terminal Version: 1.15.2874.0

### $TERM environment variable

vtpcon

### Installation

chocolatey

### How to reproduce the issue

nvim --clean
:h
CTLR-;

### Expected behavior

Nothing should happen

### Actual behavior

The current file name and cursor position are printed:
![image](https://user-images.githubusercontent.com/3659986/203880464-03214bbc-16f1-48f2-b789-bcd0cbf7de1e.png)

**Comment by 3N4N** [comment]
I can't reproduce it with v0.9.0-dev-365.

**Comment by Drllap** [comment]
Sorry, a bit of a brain dump below:

This is very wired:
* I still have the same session (Windows Terminal tab) open and I can still reproduce this there. But this doesn't happen if I open up a new tab.
* This still doesn't happen in vim in the problematic session
* The `PATH` env variable is the same in both session.
* I tried remapping `<C-;>` but doesn't appear to work:  `:map <C-;> A` (didn't go to insert mode at the end of the line)
* But I am able to remap `<C-g>`
* This still happens if I start a new powershell instance inside the problematic session
* This doesn't happen if I create a new pane in the same tab

TL;DR: This seems to be localized to one specific windows terminal session

**Comment by 3N4N** [comment]
Well, if you can't reproduce it even once, then the sensible step, it seems to me, is to just forget it ever happened.
